modules/products/static-site/static-site/storage.tf

```diff
@@ -0,0 +1,67 @@
+resource "aws_s3_bucket" "static_site" {
+  bucket = "cc-static-site-${var.product}-${var.component}"
+
+  tags = local.common_tags
```

Comment:
Reviewing to confirm that S3 tags are being applied according to the tagging model ✅